File: app/src/main/java/hr/fvlahov/barsapp/model/Item.kt
package hr.fvlahov.barsapp.model

import java.io.Serializable

@kotlinx.serialization.Serializable
data class Item(
    var id: Int,
    var name: String,
    var minSupply: Int,
    var currentSupply: Int,
    var unitOfMeasurement: UnitOfMeasurement
) : Serializable {

}

File: Web/Bar_Web_API/Bar_Web_API/Model/Item.cs
namespace Bar_Web_API.Model
{
    public class Item
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public UnitOfMeasurement UnitOfMeasurement { get; set; }
        public int MinSupply { get; set; }
        public int CurrentSupply { get; set; }
    }
}